Assets/HoloToolkit/Utilities/Plugins/README.md

```diff
@@ -0,0 +1 @@
+This folder is for the Externals.zip that should alway be up to date.
```

Comment:
This was just a placeholder file I meant to delete.

But the purpose of the zip file is so that users have all the assets they need when they import the asset package the first time. The external binaries are in the zip file and are only unpacked when users opt to use the sharing services in the HoloToolkit/Configure/Apply Project Settings menu.